examples/optimisation/nonlinearly_constrained_problem.ex.py

```diff
@@ -0,0 +1,86 @@
+# %%
```

Comment:
Needs this block from https://github.com/Fusion-Power-Plant-Framework/bluemira/tree/develop/examples#maintaining-the-examples
```suggestion
# ---
# jupyter:
#   jupytext:
#     cell_metadata_filter: tags,-all
#     notebook_metadata_filter: -jupytext.text_representation.jupytext_version
#     text_representation:
#       extension: .py
#       format_name: percent
#       format_version: '1.3'
#   kernelspec:
#     display_name: Python 3 (ipykernel)
#     language: python
#     name: python3
# ---

# %% tags=["remove-cell"]
```

Reply:
Done, I think